tile/tile.yml

```
@@ -622,6 +631,9 @@ packages:
         https_proxy: (( .properties.https_proxy_url.value ))
         http_proxy: (( .properties.http_proxy_url.value ))
         no_proxy: (( .properties.no_proxy.parsed_strings ))
+      cloud_foundry_api:
```

cloud_foundry_bbs no ?
```suggestion
      cloud_foundry_bbs:
```